freqtrade/rpc/telegram.py

```diff
@@ -1677,3 +1678,26 @@ def _send_msg(self, msg: str, parse_mode: str = ParseMode.MARKDOWN,
                 'TelegramError: %s! Giving up on that message.',
                 telegram_err.message
             )
+
+    @authorized_only
+    def _changemarketdir(self, update: Update, context: CallbackContext) -> None:
+        """
+        Handler for /marketdir.
+        Updates the bot's market_direction
+        :param bot: telegram bot
+        :param update: message update
+        :return: None
+        """
+        if context.args and len(context.args) == 1:
+            new_market_dir = context.args[0]
+            match new_market_dir:
```

Comment:
You can't use match on freqtrade for now (probably not for years).

We do support python 3.8-3.10 - match was only added in 3.10, so this will fail on all prior versions (which tests confirm as "invalid syntax" errors ...